packages/medusa/src/loaders/feature-flags/index.ts

```diff
@@ -40,12 +42,31 @@ export default (
     let from
     if (isDefined(process.env[flagSettings.env_key])) {
       from = "environment"
-      flagConfig[flagSettings.key] = isTruthy(process.env[flagSettings.env_key])
+      const envVal = process.env[flagSettings.env_key]
+
+      // MEDUSA_FF_WORKFLOWS=createProducts,deleteProducts
+      if (envVal && envVal.split(",").length > 1) {
```

Comment:
Due to the amount of paths in this control flow, it's hard to make it pretty. I'll incorporate your suggestion although I don't think it adds much. Please remember to use `suggestion` for non-blocking things.

Reply:
this snippet remove multiple path add associate the default value without having else path as well, it also uses utils instead of custom object checks and use isString instead of splitting something that might not be a string